### DISCUSSION TITLE
PowerPoint

### DISCUSSION THREAD

**Discussion by flachboard** [fmmattioni/downloadthis#19] [Ideas]
Hello,

Is it possible to use this package to download a PowerPoint created using the officer package?

**Comment by fmmattioni** [comment]
Hi, @flachboard !

You can basically download any file using `download_file()` (https://fmmattioni.github.io/downloadthis/reference/download_file.html).

If you can save the file to your local disk using the `{officer}` package, you can use that file in this function. Let me know if that works.

Felipe

**Comment by flachboard** [comment]
Ah, sorry! I was not very clear at all. 

What I meant to ask was this: Is it possible to create an html file with a PowerPoint file embedded so that a user could download it as needed?
